### PR TITLE
Replaced instances of c.f.

### PR DESCRIPTION
**Pull request by anupj**
I'm not sure if replacing c.f. with 'compared with' works but I'll defer to the original author to review and comment. I've also suggested 'as opposed to' in two instances.

### CODE REVIEW

File: error-design/thinking-about-errors.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ Errors combine in all sorts of ways:
...
+This one is a bit different to the others. It is important to consider how a program is executing when thinking about errors. We might want to treat errors differently when testing or debugging code, as opposed to, when the code is in production. We might also treat errors differently if our program is embedded in another (either as a library, a plugin, or some kind application-in-an-application-server scenario). Likewise, if our program is the OS, errors will be treated differently to if it is an application running on top of the OS. A program might also be inside some kind of VM or container with its own error handling facilities, or it might have some kind of supervisor or observer program.
```
Comment (nrc) [inline error-design/thinking-about-errors.md]: Hmm, this is a tricky one, maybe "versus"? Not sure how clear that is

File: rust-errors/result-and-error.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ At runtime, execution will stop after the second `parse` call and the value of `
 
 ## `Option`
 
-`Option` is similar to `Result` in that it is a very common, two-variant enum type with a payload type (`Some`, c.f., `Ok`). The difference is that `Option::None` (c.f., `Result::Err`) does not carry a payload. It is, therefore, analogous to `Result<T, ()>` and there are many methods for converting between `Option` and `Result`. `?` works with `Option` just like `Result`.
+`Option` is similar to `Result` in that it is a very common, two-variant enum type with a payload type `Some` (compared with `Result::Ok`). The difference is that `Option::None` (compared with `Result::Err`) does not carry a payload. It is, therefore, analogous to `Result<T, ()>` and there are many methods for converting between `Option` and `Result`. `?` works with `Option` just like `Result`.
```
Comment (nrc) [inline rust-errors/result-and-error.md]:
```suggestion
`Option` is similar to `Result` in that it is a very common, two-variant enum type with a payload type `Some` (compare to `Result::Ok`). The difference is that `Option::None` (compared with `Result::Err`) does not carry a payload. It is, therefore, analogous to `Result<T, ()>` and there are many methods for converting between `Option` and `Result`. `?` works with `Option` just like `Result`.
```

File: error-design/thinking-about-errors.md (hunk 1 above)
Comment (anupj) [reply error-design/thinking-about-errors.md]: yeah 'versus' also works - the meaning is clear from context. In my view, the whole second part of the sentence is implied and can be omitted. 

```text
We might want to treat errors differently when testing or debugging code. We might also treat errors....
```

Comment (nrc) [reply error-design/thinking-about-errors.md]: Yeah, I think that reads well.

**Comment by anupj** [comment]
@nrc is there anything else for me to do here?

**Comment by nrc** [comment]
> @nrc is there anything else for me to do here?

Do you want to make [this change](https://github.com/nrc/error-docs/pull/12#discussion_r1056840390)?

**Comment by anupj** [comment]
@nrc done

**Review by nrc — APPROVED** [review]
Thank you!

**Pull request merged**